docs/metrics.md

```diff
@@ -84,6 +84,9 @@ The total number of VMs created by namespace and virt-api pod, since install. Ty
 ### kubevirt_vm_created_total
 The total number of VMs created by namespace, since install. Type: Counter.
 
+### kubevirt_vm_disk_allocated_size_bytes
```

Comment:
@avlitman  @machadovilaca  I think it might make sense to add this to the kubevirt_vm_resource_requests metric. Wdyt?

Reply:
I like, and few changes would be needed from the current impl